distribution/lib/Standard/Table/0.0.0-dev/src/Fixed_Width/Fixed_Width_Format.enso

```diff
@@ -65,7 +65,7 @@ type Fixed_Width_Format
 
              table = Table.input [['x', [0, Nothing, Nothing, Nothing]], ['y', [4, 1, 3, 3]]]
              layout = table : Fixed_Width_Layout
-    Fixed_Width (layout:Fixed_Width_Layout) (encoding:Encoding=Encoding.default) (row_limit:Rows_To_Read=..All_Rows) (on_invalid_rows:Invalid_Fixed_Width_Rows=..Keep_Invalid_Rows) (value_formatter:Data_Formatter|Nothing=Data_Formatter.Value)
+    Fixed_Width (layout:Fixed_Width_Layout) (encoding:Encoding=Encoding.utf_8) (row_limit:Rows_To_Read=..All_Rows) (on_invalid_rows:Invalid_Fixed_Width_Rows=..Keep_Invalid_Rows) (value_formatter:Data_Formatter|Nothing=Data_Formatter.Value)
```

Comment:
Why not the `default` encoding?
I think more likely to be in Win-1252 than utf-8 so the fallback seems sensible?